.githooks/post-checkout

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+FILEID='1gYiv3V06Q-S1MWzJpIlx6Jmf2Je-mUwL'
+TARGET='rover_description/meshes/'
+echo 'Downloading Meshes'
+
+$(which python3) -m pip install gdown
+gdown --id $FILEID
```

Comment:
don't pull it in if the files are already there, otherwise every git checkout will be slow

Reply:
Wouldn't this let the files go out-of-date? Updating the files would then require manually deleting them and then running checkout again.

Reply:
Yup, we'll need a better version control system long term but right now I'd rather deal with that then super slow git checkouts